injection/clients/dynamicclient/dynamicclient_test.go

```diff
@@ -0,0 +1,56 @@
+/*
+Copyright 2019 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package dynamicclient
+
+import (
+	"context"
+	"testing"
+
+	"k8s.io/client-go/rest"
+
+	"github.com/knative/pkg/controller"
+	"github.com/knative/pkg/injection"
+)
+
+func TestRegistration(t *testing.T) {
+	ctx := context.Background()
+
+	// Get before registration
+	if empty := Get(ctx); empty != nil {
+		t.Errorf("Unexpected informer: %v", empty)
+	}
+
+	// Check how many informers have registered.
+	inffs := injection.GetClients()
+	if want, got := 1, len(inffs); want != got {
+		t.Errorf("GetInformers() = %d, wanted %d", want, got)
```

Comment:
This block is fetching `clients` but the comment & error message say `informers`

Reply:
good catch